src/main/groovy/org/mockito/release/internal/gradle/BootstrapPlugin.java

```diff
@@ -0,0 +1,27 @@
+package org.mockito.release.internal.gradle;
+
+import org.gradle.api.Action;
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.Task;
+import org.mockito.release.internal.gradle.util.TaskMaker;
+
+/**
+ * Creates task initShipkit that all other init tasks should depend on
+ * so that running it would create all configuration needed to start Shipkit
+ */
+public class BootstrapPlugin implements Plugin<Project> {
+
+    public static final String INIT_SHIPKIT_TASK = "initShipkit";
```

Comment:
Let's use consistent naming. If the plugin is BootstrapPlugin, let's call the task BootstrapShipkit. If the plugin is InitPlugin, let's call the task "initShipkit". I think this will make our plugins easier to use.